docker/checkout.bash

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+#  Copyright (C) 2018-2020 LEIDOS.
+# 
+#  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+#  use this file except in compliance with the License. You may obtain a copy of
+#  the License at
+# 
+#  http://www.apache.org/licenses/LICENSE-2.0
+# 
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations under
+#  the License.
+
+# CARMA packages checkout script
+# Optional argument to set the root checkout directory with no ending '/' default is '~'
+
+set -exo pipefail
+
+dir=~
+while [[ $# -gt 0 ]]; do
+      arg="$1"
+      case $arg in
+            -d|--develop)
+                  BRANCH=develop
+                  shift
+            ;;
+            -r|--root)
+                  dir=$2
+                  shift
+                  shift
+            ;;
+      esac
+done
+
+if [[ "$BRANCH" = "develop" ]]; then
+      git clone https://github.com/usdot-fhwa-stol/carma-msgs.git ${dir}/src/CARMAMsgs --branch $BRANCH --depth 1
+      git clone https://github.com/usdot-fhwa-stol/carma-utils.git ${dir}/src/CARMAUtils --branch $BRANCH --depth 1    
+else
+      git clone https://github.com/usdot-fhwa-stol/carma-msgs.git ${dir}/src/CARMAMsgs --branch carma-system-3.4.0 --depth 1
+      git clone https://github.com/usdot-fhwa-stol/carma-utils.git ${dir}/src/CARMAUtils --branch carma-system-3.4.0 --depth 1
+fi
+
+# Required to build the dbw_pacifica_msgs message set.
+git clone https://github.com/NewEagleRaptor/raptor-dbw-ros.git ${dir}/src/raptor-dbw-ros --branch master --depth 1
```

Comment:
Can we lock this checkout to a specific commit? 
The current head of master seems to be https://github.com/NewEagleRaptor/raptor-dbw-ros/commit/f50f91cd88ad27b2ce05bab1f8ff780931c41475

At a minimum I think we should add the commit hash to the comment if changing the script is too risky

Reply:
Looks like that's the commit we must have been building candidate images on since it's been there since sept 14. I'll update the script